dev-games/goatee/metadata.xml

```diff
@@ -1,21 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-	<maintainer type="project">
```

Comment:
Did you ask someone on games@?

Reply:
I did not - should I?

Copied this over from `::haskell`, so I thought it was appropriate to leave it with `haskell@`

Reply:
You're dropping them as a maintainer, so yes ;)

Reply:
This is moot now, as @trofi has bumped the package already on `master`. Closing this, and removing from this PR.